x-pack/plugins/secops/public/components/timeline/body/renderers/source_dest_ip.test.tsx

```diff
@@ -0,0 +1,44 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import euiDarkVars from '@elastic/eui/dist/eui_theme_dark.json';
+import toJson from 'enzyme-to-json';
+import { noop } from 'lodash/fp';
+import * as React from 'react';
+import { Provider } from 'react-redux';
+import { ThemeProvider } from 'styled-components';
+
+import { DragDropContext } from 'react-beautiful-dnd';
+import { mountWithIntl } from 'test_utils/enzyme_helpers';
```

Comment:
[Optional](https://github.com/elastic/kibana/pull/29311#discussion_r252078544): Separate regular imports and dotted local ones. Should we go ahead and add these [separate groupings](https://palantir.github.io/tslint/rules/ordered-imports/) to our `tsconfig.json`?

Reply:
Thanks. Yea I think we should add that tslint option.